### PR TITLE
Fix lint conteiners reducers and services

### PR DESCRIPTION
**Pull request by LesiaM-T**
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/ita-social-projects/EventsExpress-Frontend/projects/1)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

fixed linter plroblems in some containers, reducers and services
## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by VadymKharyton — COMMENTED** [review]
Please change PR base branch, because there are a lot of changes, and I'm not able to identify exactly your changes

**Comment by LesiaM-T** [comment]
> Please change PR base branch, because there are a lot of changes, and I'm not able to identify exactly your changes

Sorry, made it by mistake. Now changed the base.

File: package.json [hunk 1]
```diff
@@ -3,110 +3,110 @@
 	"version": "0.1.0",
 	"private": true,
 	"dependencies": {
```
Comment (Vladosik12333) [inline package.json:5]: Delete changes in package.json

File: src/containers/unitsOfMeasuring/unitOfMeasuring-item.js [hunk 2]
```diff
@@ -137,20 +137,12 @@ UnitOfMeasuringItemWrapper.defaultProps = {
...
   editedUnitOfMeasuring: null,
   allCategories: {},
-  editСancel: () => {},
-  save_unitOfMeasuring: () => {},
-  delete_unitOfMeasuring: () => {},
-  setUnitOfMeasuringEditedDispatch: () => {},
 };
 
 UnitOfMeasuringItemWrapper.propTypes = {
   item: PropTypes.object,
```
Comment (Vladosik12333) [inline src/containers/unitsOfMeasuring/unitOfMeasuring-item.js:143]: You have propTypes on 96 line. Why you use two propTypes, you need only one. Transfer these propTypes in propTypes on 96 line.

File: src/reducers/hub.js [hunk 3]
```diff
@@ -11,9 +11,7 @@ const reducer = (state = initialState.hubConnections, action) => {
         chatHub: action.payload,
       };
     case EVENT_WAS_CREATED:
-      state.chatHub
-        .invoke("EventWasCreated", action.payload)
-        .catch(err => console.error(err));
+      state.chatHub.invoke("EventWasCreated", action.payload).catch(err => err);
```
Comment (Vladosik12333) [inline src/reducers/hub.js:14]: Write TODO: Make error handling.

File: src/reducers/root.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import { routerReducer } from "react-router-redux";
 import { reducer as formReducer } from "redux-form";
-import { reducer as LoginReducer } from "./login";
+import * as LoginReducer from "./login";
```
Comment (Vladosik12333) [inline src/reducers/root.js:3]: Do you sure this work? If LoginReducer return not default, you will need `{... as ...}`.

File: src/services/EventsExpressService.js [hunk 5]
```diff
@@ -32,14 +32,14 @@ export default class EventsExpressService {
   };
 
   setResource = async (Url, data) => {
-    const call = (Url, data) =>
-      fetch(this.baseUrl + Url, {
+    const call = (url, Data) =>
```
Comment (Vladosik12333) [inline src/services/EventsExpressService.js:35]: Why you write Data from big letter? If you need another parameters from function setResource, you will write callUrl and callData.

File: src/services/EventsExpressService.js [hunk 6]
```diff
@@ -32,14 +32,14 @@ export default class EventsExpressService {
   };
 
   setResource = async (Url, data) => {
```
Comment (Vladosik12333) [inline src/services/EventsExpressService.js:34]: `Url` must be `url`.

File: src/services/InventoryService.js [hunk 7]
```diff
@@ -37,7 +37,8 @@ export default class InventoryService {
     );
 
   setUsersInventoryDelete = async data => {
-    data.quantity = 1;
+    const dataCopy = data;
+    dataCopy.quantity = 1;
     await baseService.setResource(`UserEventInventory/Delete`, data);
```
Comment (Vladosik12333) [inline src/services/InventoryService.js:42]: If you write dataCopy instead of data, you will need write dataCopy in here.

**Review by Vladosik12333 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (LesiaM-T) [reply package.json:5]: i need to add "react-router"

Comment (Vladosik12333) [reply package.json:5]: Ohh.. yes.

File: src/services/EventsExpressService.js (hunk 6 above)
Comment (LesiaM-T) [reply src/services/EventsExpressService.js:34]: specify, please, wich row you mean?

Comment (Vladosik12333) [reply src/services/EventsExpressService.js:34]: 34

File: src/services/EventsExpressService.js [hunk 8]
```diff
@@ -22,7 +22,7 @@ export default class EventsExpressService {
   };
 
   getPhoto = async Url => {
```
Comment (Vladosik12333) [inline src/services/EventsExpressService.js:24]: Url must be lower case.
Write `url` instead `Url` in full file and rename some vars (example: url -> callUrl).

File: src/services/InventoryService.js [hunk 9]
```diff
@@ -37,8 +37,9 @@ export default class InventoryService {
     );
 
   setUsersInventoryDelete = async data => {
-    data.quantity = 1;
-    await baseService.setResource(`UserEventInventory/Delete`, data);
+    const dataCopy = data;
```
Comment (Vladosik12333) [inline src/services/InventoryService.js:40]: Must write `const dataCopy = {...data}`

**Review by Vladosik12333 — APPROVED** [review]

**Pull request merged**